library/src/scala/compiletime/package.scala

```diff
@@ -18,6 +16,15 @@ package object compiletime {
    */
   erased def erasedValue[T]: T = ???
 
+  /** Used as the initializer of a mutable class or object field, like this:
+   *
+   *    var x: T = uninitialized
+   *
+   *  This signifies that the field is not initialized on its own (On the JVM
+   *  it is still bulk-initialized to a zero bitfield).
```

Comment:
Too low-level an explanation?  Is it obvious or known enough that this mean `null`/`false`/`0`/`0L`/etc?

Reply:
Also it's not just on the JVM. Scala.js and Scala Native have to replicate that behavior, obviously.